theano/gof/destroyhandler.py

```diff
@@ -788,24 +790,27 @@ def fast_destroy(self, app):
         - Allow sequence of view.
         - But don't allow to destroy view
         """
-        return
         dm = getattr(app.op, 'destroy_map', None)
         if not dm:
             return
-        inputs = sum(dm.values())  # list of app's destroyed inputs
+        # inputs = sum(dm.values())  # list of app's destroyed inputs
+        inputs = dm.values()[0]
```

Comment:
Thanks, I fixed it. The items inside ```dm.values()```  are them selves lists. In the new version the ```inputs``` is a list of all the inputs that have been destroyed.